iohub/ngff.py

```diff
@@ -900,6 +915,41 @@ def update_channel(self, chan_name: str, target: str, data: ArrayLike):
         ortho_sel[ch_ax] = ch_idx
         img.set_orthogonal_selection(tuple(ortho_sel), data)
 
+    def initialize_pyramid(self, levels: int) -> None:
+        """
+        Initializes the pyramid arrays with a down scaling of 2 per level.
+        Decimals shapes are rounded up to ceiling.
+        Scales metadata are also updated.
+
+        Parameters
+        ----------
+        levels : int
+            Number of down scaling levels, if levels is 1 nothing happens.
+        """
+        array = self.data
+        for level in range(1, levels):
+            factor = 2**level
+            shape = array.shape[:2] + _scale_integers(array.shape[2:], factor)
```

Comment:
An [NGFF image](https://ngff.openmicroscopy.org/0.4/#multiscale-md) may not have all 5 dimensions. See:
https://github.com/czbiohub/iohub/blob/07c7925a735773ab2a652a6177483a7863ef4ea7/iohub/ngff.py#L721-L746